src/style/CampaignPage.less

```diff
@@ -472,4 +478,16 @@ body {
 			overflow-y: hidden;
 		}
     }
+
+	.wider-impact-quote {
+		background: #5BA6BD;
+		font-weight: 900;
+		font-size: 1.5rem;
+		font-family: "Montserrat", sans-serif !important;
```

Comment:
Hello @dodo721 We try to avoid local font css, as it leads to inconsistent fonts. Is this equivalent to one of the h1 - h5 header styles? If so, you could use that tag, or use a .h2 etc css class (which Bootstrap defines to be the same as the h tags).

Reply:
I couldn't get it to work with header tags/classes without a pile of other overrides and !importants on top - however I found the !important rules I had put in are unnecessary, so I have removed them and they should now be just as overrideable as any header.